site/en/docs/crux/methodology/index.md

```diff
@@ -0,0 +1,565 @@
+</tr>
+<tr>
+<td>slow-2g</td>
+<td>2000ms</td>
+<td>50 Kbps</td>
+<td>The network is suited for small transfers only such as text-only pages.</td>
+</tr>
+<tr>
+<td>2g</td>
+<td>1400ms</td>
+<td>70 Kbps</td>
+<td>The network is suited for transfers of small images.</td>
+</tr>
+<tr>
+<td>3g</td>
+<td>270ms</td>
+<td>700 Kbps</td>
+<td>The network is suited for transfers of large assets such as high resolution images, audio, and SD video.</td>
+</tr>
+<tr>
+<td>4g</td>
+<td>0ms</td>
+<td>∞</td>
+<td>The network is suited for HD video, real-time video, etc.</td>
+</tr>
+</tbody>
+</table></div>
+
+### Form Factor {: #form-factor-dimension }
+
+The form factor dimension allows you to query against three separate form factors:
+
+- Mobile (`PHONE`)
+- Tablet (`TABLET`)
+- Desktop (`DESKTOP`)
+
+Form factor is inferred from the device [user-agent string](/docs/multidevice/user-agent/).
+
+### Country {: #country-dimension }
+
+The country dimension was [added to CrUX in 2018](/blog/crux-2018-01/). The term "country" is used loosely, as some geographic areas are politically disputed. Values in the country dimension are inferred from users' IP addresses and represented as two-letter country codes as defined by [ISO 3166-1](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2).
+
+Country-level datasets are provided in addition to the global dataset, with the standard eligibility requirements applied at a country level. A table is [provided for each country](../bigquery/#schema-country-summary), as well as [summary tables](../bigquery/#schema-raw-tables) which include the country code as a column.
+
+### Optional dimensions {: #optional-dimensions}
+
+As of the [May 2022](../release-notes/#202205) release, the CrUX dataset supports optional dimensions. Previously, a form factor and effective connection type (ECT) combination must have independently met the _sufficiently popular_ criterion or else it would be excluded from the page or origin record. With this feature, experiences on different ECTs could be combined by their common form factor, and the corresponding ECT value will be `NULL`. Experiences on different form factors may also be combined, and both the ECT and form factor values will be `NULL`.
+
+Previously, form factor and effective connection type were required columns in our BigQuery tables. This meant that when we did not have sufficient coverage to express the histogram densities in the specific rows (e.g., form factor = phone, effective connection type = 2G), we were dropping the entire origin from the dataset. With optional dimensions, we made the form factor and effective connection type optional (NULLABLE) and therefore, we're now able to publish overall histogram densities in such cases; that is, we may set the effective connection type value to NULL indicating "all effective connection types", or we may set both effective connection type and form factor to NULL indicating "all effective connection types" and "all form factors".
```

Comment:
I think I prefer the more verbose version, @rviscomi thoughts?

Reply:
Verbose version SGTM if that's what you think is clearer for readers.

@tunetheweb we're only referring to the public dataset here.

Reply:
Yeah I just think saying "Previously, form factor and effective connection type were required columns in our BigQuery tables" is confusing as most people only know the public dataset that is updated once a month.

This change also affects the daily API, and yes the reason that that is affected too, is that they are required columns in our private BigQuery dataset. But that's the point that I think is confusing.

So I'm suggesting dropping any mention of "required columns" or "BigQuery tables" to avoid that confusion.

Reply:
Synced with Simon on this. It's ok to leave as-is.